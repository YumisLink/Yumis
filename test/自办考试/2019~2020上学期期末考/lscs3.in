10 10
10526 17976 24619 16001 9219 19711 13273 28009 2676 8081 