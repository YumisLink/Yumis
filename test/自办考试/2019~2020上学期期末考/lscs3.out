3508 3518